src/main/resources/lib/enonic/polyfill-react4xp/nashornPolyfills.ts

```diff
@@ -13,6 +14,7 @@
 import Map from 'core-js-pure/actual/map';
 import Set from 'core-js-pure/actual/set';
 import Symbol from 'core-js-pure/actual/symbol';
+import {polyfillTimers} from './timers';
```

Comment:
polyfillNode runs for nashorn, too.
We don't need to polyfill timers twice

Reply:
Ah I probably confused myself because mavenLocal wasn't in build.gradle when I started testing, soo I assumed that one was for Graal and the other for Nashorn :) Easy fix though.